db/db.json

```diff
@@ -3,16 +3,82 @@
     {
       "id": 1,
       "firstName": "John",
-      "lastName": "Carmack",
+      "lastName": "Carmock",
       "street": "10 Keen Street",
       "city": "London"
     },
     {
       "id": 2,
       "firstName": "Grace",
-      "lastName": "Hopper",
+      "lastName": "Harper",
       "street": "32 Deebug Road",
       "city": "London"
+    },
+    {
+      "firstName": "Alex",
+      "lastName": "111",
+      "street": "Via Adige, 47",
+      "city": "Bari",
+      "id": 3
+    },
+    {
+      "firstName": "Alex",
+      "lastName": "jennings",
+      "street": "Via Adige, 47",
+      "city": "Bari",
+      "id": 4
+    },
+    {
+      "firstName": "Alex",
+      "lastName": "Jennings",
+      "street": "Via Adige, 47",
+      "city": "Bari",
+      "email": "labassottaarezzo@gmail.com",
+      "twitter": "",
+      "linkedIn": "labass",
+      "TruthSocial": "labaass",
+      "id": 5
+    },
+    {
+      "firstName": "dfa",
+      "lastName": "dasf",
+      "street": "adsf",
+      "city": "asdf",
+      "email": "asdfa@dfas",
+      "truthSocial": "gfs",
+      "linkedIn": "afkj",
+      "TruthSocial": "alkjf",
+      "id": 6
+    },
+    {
+      "firstName": "fad",
+      "lastName": "afdf",
+      "street": "asdf",
+      "city": "adf",
+      "email": "aldfjl@kfjal.com",
+      "truthSocial": "dfgdaf",
+      "linkedIn": "ldakfj",
+      "id": 7
+    },
```

Comment:
You added new contacts to test if it' working. 
Maybe you can delete some of the extra contacts to clean your code? 
This is something I would do.